.github/workflows/test_security.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   integ-test-with-security-linux:
     strategy:
       matrix:
-        java: [11, 17, 21]
+        java: [11, 17]
```

Comment:
We should keep testing on 21, but change it to 21.0.1 (effectively backporting #429).

I'm not sure why we don't test JDK 21 on 2.x, though.  It's the latest LTS JDK and 2.x is what we're actually releasing.

Reply:
I had thought there was a reason we excluded JDK 21 from the 2.x branch, though I'm not too sure. Looking at the original PRs I don't see any mention of this. I can go ahead and change this to 21.0.1

Reply:
OpenSearch tests on 2.x (2.12.0) are using JDK 21.0.1.